TLM/TLM/Manager/Impl/AdvancedParkingManager.cs

```diff
@@ -2418,7 +2417,7 @@ bool citizenDebug
                                      VehicleRestrictionsMode.Configured)
                                  & ExtVehicleType.PassengerCar) != ExtVehicleType.None)
                             {
-                                if (CustomPassengerCarAI.FindParkingSpaceRoadSide(
+                                if (Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection.FindParkingSpaceRoadSide(
```

Comment:
Style: This would read shorter with a const, something like `const pcai_conn = Constants.ManagerFactory.GameConnectionManager.PassengerCarAIConnection;`, if it accesses that const more than once, it'll even improve the performance.

Reply:
Good point!